.github/workflows/build-image.yml

```diff
@@ -8,6 +8,33 @@ jobs:
   bashls:
     name: Push bashls to Docker Hub
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - context: servers/bashls
+            tags: lspcontainers/bash-language-server:1.17.0
+
+          - context: servers/dockerls
+            tags: lspcontainers/docker-language-server:0.4.1
+
+          - context: servers/gopls
+            tags: lspcontainers/gopls:0.6.11
+
+          - context: servers/pyright
+            tags: lspcontainers/pyright-langserver:1.1.137
+
+          - context: servers/rust_analyzer
+            tags: lspcontainers/rust-analyzer:2021-05-03
+
+          - context: servers/sumneko_lua
+            tags: lspcontainers/lua-language-server:1.20.5
+
+          - context: servers/tsserver
+            tags: lspcontainers/typescript-language-server:0.5.1
+
+          - context: servers/yamlls
+            tags: lspcontainers/yaml-language-server:0.18.0
```

Comment:
would it be worthwhile to rename this matrix keys to something a little more self-descriptive?

like:

```
  - subdirectory: servers/pyright
    image_name: lspcontainers/gopls:0.6.11
```

Reply:
I think at the moment to avoid confusion and the need to make more variables to have a 1:1 reference here. If we find ourselves needing duplicate keys we can do things like `with_context` and `foo_context`.